raiden/accounts.py

```diff
@@ -10,10 +10,12 @@
 def find_datadir():
     home = os.path.expanduser("~")
     datadir = None
-    if sys.platform.startswith('linux'):
+    if sys.platform.startswith('linux') or sys.platform.startswith('FreeBSD'):
```

Comment:
Hmm I don't see why the current one is bad except for the additional or check if you got FreeBSD but sure I can change it.